libs/blog-bff/shared/util-wp/src/lib/wp-client.ts

```diff
@@ -16,14 +16,26 @@ export class WPRestClient {
 
   get<T>(url: string, params: Record<string, string>): Promise<WPRespone<T>> {
     const searchParams = new URLSearchParams(params);
-    return this.request(`${url}?${searchParams.toString()}`, { method: 'GET' });
+    return this.request(`/wp-json/wp/v2/${url}?${searchParams.toString()}`, {
+      method: 'GET',
+    });
+  }
+
+  postEmail<T>(body: T): Promise<WPRespone<T>> {
```

Comment:
shouldn't be put here. it's a base class to make wp requests.
Either rename to `post`, or create a separate class.

Reply:
Renamed